reference/5-api-publisher/API-Connection-Management.md

```diff
@@ -0,0 +1,64 @@
+| `/ed-fi/apiPublisher/connections/_{connectionName}_/exclude`                                  | String       | (_Optional_) For _source_ API connections, the resources (and their dependents) to NOT publish to the target. The value is defined using the same format as with `--include` (see above). The Ed-Fi API Publisher will also evaluate and automatically exclude all dependent resources of the excluded resources (using the dependency metadata exposed by the target API). This will ensure (barring misconfigured authorization metadata or data policies) that data can be successfully published to the target API.                                                                                                                                                                                                                     |
+| `/ed-fi/apiPublisher/connections/_{connectionName}_/excludeOnly`                              | String       | (_Optional_) For _source_ API connections, the specific resources to skip publishing to the target (dependent resources will still be published). The value is defined using the same format as with `--include` (see above).<br/><br/> NOTE: Use caution when publishing without automatically including all dependencies.                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| <nobr>`/ed-fi/apiPublisher/connections/_{connectionName}_/lastChangeVersionsProcessed`</nobr> | String       | _(Optional)_ For _source_ API connections, contains a JSON object, keyed by target API name, that indicates the last change version successfully published from the source to the target. This value is automatically created/updated by the Ed-Fi API Publisher after successfully completing the publishing process.                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
This table ends up scrolling past the visible screen. This content might not be a good fit for a table in this sort of display. Might be better as a bulleted list.

![image](https://github.com/user-attachments/assets/769cb977-7406-47c4-9b79-0c40a6f5be41)

Reply:
I reduced the size of the parameters now you can see the 3 columns..

Reply:
Yes, that helps.